### DISCUSSION TITLE
Missing option for hardware acceleration?

### DISCUSSION THREAD

**Discussion by Hynpothalamus** [JakubMelka/PDF4QT#199] [Q&A] [answered]
Greetings. 

According to readme there is a feature for hardware accelerated rendering. But i can't find the setting for it and taskmgr shows no sign of GPU utilization. I haven't encountered any rendering error and the log is empty. Did I missed something?

I tried manually assigning both my iGPU and dGPU to the viewer/editor and didn't make a change.

I'm using a 12600k with UHD graphics 770 and a Quadro P4000 which should provide complete support for openGL and the driver is up-to-date (Jul.9 2024).

PS: Although i can't get the hardware acceleration but performance of Blend2D is already very impressive compared to Acrobat.

![20240725121440](https://github.com/user-attachments/assets/cd881a7e-cf01-4f12-b38a-9d6c4ee3c662)

**Comment by JakubMelka** [comment, marked as answer]
Hello @Hynpothalamus , I have disabled hardware acceleration because, in most cases (with the exception of some documents containing large images), Blend2D outperformed hardware-accelerated rendering. Moreover, OpenGL support is deprecated in Qt6, which uses a different approach to hardware acceleration compared to Qt5. Additionally, hardware acceleration is usually not supported on virtual machines.

**Reply by Hynpothalamus** [reply]
Thank you for the detailed explaination. Just as informative as the text in the software!
